backends/smt2/smtio.py

```diff
@@ -203,15 +203,14 @@ def setup(self):
                 print('timeout option is not supported for mathsat.')
                 sys.exit(1)
 
-        if self.solver == "boolector":
+        if self.solver in ["boolector", "bitwuzla"]:
             if self.noincr:
-                self.popen_vargs = ['boolector', '--smt2'] + self.solver_opts
+                self.popen_vargs = [self.solver, '--smt2'] + self.solver_opts
             else:
-                self.popen_vargs = ['boolector', '--smt2', '-i'] + self.solver_opts
+                self.popen_vargs = [self.solver, '--smt2', '-i'] + self.solver_opts
             self.unroll = True
             if self.timeout != 0:
-                print('timeout option is not supported for boolector.')
-                sys.exit(1)
+                self.popen_vargs.append('--time=%d' % self.timeout);
```

Comment:
this doesn't quite work properly — while the timeout will cause boolector/bitwuzla to terminate, it'll terminate with the following output:

```
[btor>main] ALARM TRIGGERED: time limit 1 seconds reached                                                                                                                                              
unknown                                                                                                                                                                                                
```

this, unfortunately, won't be properly picked up as "timeout" status by the wrapper code below